manifests/init.pp

```diff
@@ -3,29 +3,33 @@
 # Manage RANCID - http://www.shrubbery.net/rancid/
 #
 class rancid (
-  $filterpwds         = 'ALL', # yes, no, all
-  $nocommstr          = 'YES', # yes or no
-  $maxrounds          = '4',
-  $oldtime            = '4',
-  $locktime           = '4',
-  $parcount           = '5',
-  $maildomain         = undef,
-  $groups             = [ 'routers', 'switches', 'firewalls' ],
-  $devices            = undef,
-  $packages           = 'USE_DEFAULTS',
-  $rancid_config      = 'USE_DEFAULTS',
-  $rancid_path_env    = 'USE_DEFAULTS',
-  $homedir            = 'USE_DEFAULTS',
-  $logdir             = 'USE_DEFAULTS',
-  $user               = 'USE_DEFAULTS',
-  $group              = 'USE_DEFAULTS',
-  $shell              = 'USE_DEFAULTS',
-  $cron_d_file        = '/etc/cron.d/rancid',
-  $cloginrc_content   = 'USE_DEFAULTS',
-  $show_cloginrc_diff = true,
+  $filterpwds          = 'ALL', # yes, no, all
+  $nocommstr           = 'YES', # yes or no
+  $maxrounds           = '4',
+  $oldtime             = '4',
+  $locktime            = '4',
+  $parcount            = '5',
+  $maildomain          = undef,
+  $groups              = [ 'routers', 'switches', 'firewalls' ],
+  $devices             = undef,
+  $packages            = 'USE_DEFAULTS',
+  $rancid_config       = 'USE_DEFAULTS',
+  $rancid_path_env     = 'USE_DEFAULTS',
+  $homedir             = 'USE_DEFAULTS',
+  $logdir              = 'USE_DEFAULTS',
+  $user                = 'USE_DEFAULTS',
+  $group               = 'USE_DEFAULTS',
+  $shell               = 'USE_DEFAULTS',
+  $cron_d_file         = '/etc/cron.d/rancid',
+  $cloginrc_content    = 'USE_DEFAULTS',
+  $show_cloginrc_diff  = true,
+  $vcs                 = 'USE_DEFAULTS',
+  $vcsroot             = 'USE_DEFAULTS',
+  $manage_vcs_packages = true,
```

Comment:
The package part is new, so this should actually default to `false`. By defaulting to `true` it will almost certainly cause issues for current users related to duplicate resources.